### ISSUE TITLE
@property annotation not implemented in 1.3.0 release

### ISSUE THREAD

**Issue by jcheron** [php-annotations/php-annotations#129]
With the last stable release (1.3.0), 
Using the standard `@property` annotation generates an exception:
```
PHP Fatal error:  Uncaught mindplay\annotations\AnnotationException: Annotation type 'mindplay\annotations\standard\PropertyAnnotation' does not support PHP-DOC style syntax (because it does not implement the mindplay\annotations\IAnnota
tionParser interface)
```
I made a commit https://github.com/php-annotations/php-annotations/commit/8a07adadcec7dcab809700378982d0d035617bb6 with @aik099  integrating the implementation of PropertyAnnotation.
Is it possible to create a new stable release incorporating these changes?

Thanks

**Comment by aik099** [comment]
Yes, it's possible. While attempting to create a release I've noticed, that `CHANGELOG.md` wasn't updated by PR's, that were created since last release. If you could send a PR to update it, then it would be great.

Here is a list of PR's in question:

1. #116
2. #120
3. #121
4. #122

**Comment by aik099** [comment]
I've released 1.3.1 version with your help: https://github.com/php-annotations/php-annotations/releases/tag/1.3.1

**Comment by jcheron** [comment]
@aik099, thank you for responsiveness !
